hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/CompactStreamSerializerTest.java

```diff
@@ -469,7 +472,9 @@ public void testFieldOrder() throws IOException {
 
         SchemaWriter writer = new SchemaWriter("typeName");
 
-        ReflectiveCompactSerializer reflectiveCompactSerializer = new ReflectiveCompactSerializer();
+        CompactStreamSerializer compactStreamSerializer = mock(CompactStreamSerializer.class);
+        when(compactStreamSerializer.canBeSerializedAsCompact(any())).thenReturn(true);
+        ReflectiveCompactSerializer reflectiveCompactSerializer = new ReflectiveCompactSerializer(compactStreamSerializer);
```

Comment:
Can `CompactTestUtil.getSchemasFor` be used instead of these lines?

Reply:
I mean maybe a new method could be introduced for these lines, since it was repeated 3 times (also in getSchemasFor). Just a nit.